spec/support/capybara_drivers.rb

```diff
@@ -1,46 +1,36 @@
 # frozen_string_literal: true
 
 module CapybaraDrivers
-  def sign_in(user: nil, driver:)
-    assign_current_driver(driver)
+  def sign_in(user: nil)
     OmniAuth.config.test_mode = true
     OmniAuth.config.mock_auth[:psu] = mock_auth_hash(user) if user.present?
   end
 
   private
 
-    def assign_current_driver(driver)
-      register_driver(driver) unless Capybara.drivers.include?(driver)
-      Capybara.current_driver = driver
-    end
-
-    def register_driver(driver)
-      Capybara.register_driver driver do |app|
-        capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(chromeOptions: chrome_options)
-        Capybara::Selenium::Driver.new(app, browser: :chrome, desired_capabilities: capabilities)
-      end
-    end
-
-    def chrome_options
-      { args: ['no-sandbox', 'disable-gpu', 'window-size=1024,768', 'single-process'] }
-    end
-
     def mock_auth_hash(user)
       build :psu_oauth_response, access_id: user.access_id
     end
 end
 
 RSpec.configure do |config|
   config.before(type: :feature) do |example|
-    driver = example.metadata.fetch(:with_driver, Capybara.current_driver)
-
     if example.metadata.key?(:with_user)
-      sign_in(user: send(example.metadata.fetch(:with_user)), driver: driver)
+      sign_in(user: send(example.metadata.fetch(:with_user)))
     else
-      sign_in(driver: driver)
+      sign_in
     end
   end
 
   config.include CapybaraDrivers, type: :feature
   config.include ActionView::RecordIdentifier, type: :feature
 end
+
+Capybara.register_driver :selenium_chrome_headless do |app|
```

Comment:
@rschenk I'm not sure this block is even necessary. `: selenium_chrome_headless` is already present in the list of available drivers, so the only line that may be needed is line 36. This might change if we need to specify a window size. We did need to do this in some other tests to verify that UI elements were visible in a tablet and phone-sized page, but I'd rather see us put this in when they're needed. I'm also not convinced the other options actually do anything.

Reply:
Cool. In that case, I suppose we could always register our own custom drivers, like `:selenium_chrome_tablet`, `:selenium_chrome_phone`, etc etc